frontend/src/components/authForm/validate.js

```diff
@@ -0,0 +1,41 @@
+export const validateRegistration = (values) => {
+	const errors = {};
+	
+	if (!values.name) {
+		errors.name = 'Required';
+	}
+	if (!values.email) {
+		errors.email = 'Required';
+	} else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values.email)){
```

Comment:
you can do `Regex` here. For now, im okay with the implementation but for future improvement lang.
you can try it [here](https://www.freecodecamp.org/news/javascript-triple-equals-sign-vs-double-equals-sign-comparison-operators-explained-with-examples/#:~:text=Double%20Equals%20(%20%3D%3D%20)%20checks%20for,does%20not%20perform%20type%20coercion.)